etc/vite-plugin-config.ts

```diff
@@ -145,19 +147,31 @@ function parseConfig(raw: ConfigRaw): ConfigParsed {
     return token
   }
 
+  const parseURL = (url: string): string => {
```

Comment:
```suggestion
  const trimTrailingSlash = (str: string): string => {
```

It would make more sense, I think.